core-bundle/src/EventListener/DataContainer/AdjustAutoforwardSubpaletteListener.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener\DataContainer;
+
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\DataContainer;
+use Contao\PageModel;
+
+class AdjustAutoforwardSubpaletteListener
+{
+    /**
+     * @var ContaoFramework
+     */
+    private $framework;
+
+    public function __construct(ContaoFramework $framework)
+    {
+        $this->framework = $framework;
+    }
+
+    public function __invoke(DataContainer $dc): void
+    {
+        if ('tl_page' !== $dc->table || !$dc->id) {
+            return;
+        }
+
+        /** @var PageModel $pageAdapter */
+        $pageAdapter = $this->framework->getAdapter(PageModel::class);
+
+        if (!($page = $pageAdapter->findByPk($dc->id)) || 'error_403' === $page->type) {
+            return;
+        }
+
+        $GLOBALS['TL_DCA'][$dc->table]['subpalettes']['autoforward'] = str_replace(',redirect', '', $GLOBALS['TL_DCA'][$dc->table]['subpalettes']['autoforward']);
```

Comment:
```suggestion
        $GLOBALS['TL_DCA']['tl_page']['subpalettes']['autoforward'] = str_replace(',redirect', '', $GLOBALS['TL_DCA']['tl_page']['subpalettes']['autoforward']);
```

Since we have decided against making this a universal listener, we might as well hardcode the table name here. Just like we have hardcoded `PageModel::class` above.